boards/arm/rcar_h3ulcb/rcar_h3ulcb_cr7.yaml

```diff
@@ -11,6 +11,7 @@ supported:
   - gpio
   - clock_control
   - uart
+  - pwm
```

Comment:
If you do not support the pwm test suite (according to your PR message), you should not mark it as supported in your board's yaml file. Otherwise, you will get errors on the pwm suite every time you run twister on your board.
So I think that you should mark this suite as ignored for now before marking it as supported later if you find a way to patch either your driver or the test suite.

Reply:
Good point here. I will rework this until the test could be passed with success.